src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +22,20 @@
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl(
+                new CountryDaoImpl(sessionFactory)); // TODO: initialize this instance
         countryService.add(usa);
 
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
+        ActorService actorService = new ActorServiceImpl(
+                new ActorDaoImpl(sessionFactory)); // TODO: initialize this instance
```

Comment:
The `ActorService` instance is being initialized correctly with `ActorServiceImpl` and `ActorDaoImpl`. However, the TODO comment suggests that this initialization needs to be verified or completed. Ensure that the `ActorServiceImpl` constructor is correctly implemented to accept an `ActorDaoImpl` instance.